calico/networking/configuring/advertise-service-ips.mdx

```diff
@@ -216,22 +216,17 @@ To install the MetalLB controller for allocating addresses, perform the followin
 
    However, do not install the `metallb-system/speaker` component. The speaker component also attempts to establish BGP sessions on the node, and will conflict with Calico.
 
-1. Configure MetalLB to provision addresses by creating the following config map, replacing `x.x.x.x/16` with the CIDR given to {{prodname}} in the steps above.
+1. Configure MetalLB to provision addresses by creating the following IPAddressPool, replacing `x.x.x.x/16` with the CIDR given to {{prodname}} in the steps above. Please note IPAddressPool require Metallb `0.13+`.
```

Comment:
```suggestion
1. Configure MetalLB to provision addresses by creating the following IPAddressPool, replacing `x.x.x.x/16` with the CIDR given to {{prodname}} in the steps above. Please note IPAddressPool requires Metallb `0.13+`.
```

Reply:
Almost there. After fixing above sentence, please copy the changes to these folders as well:

calico-versioned-docs
   3.26
   3.25

calico-enterprise-docs
   3.19 (root)
calico-enterprise-versioned-docs
   3.18
   3.17

calico-cloud
  3.19 (root)
calico-cloud-versioned-docs
  3.18
  3.17